packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -12,7 +12,7 @@ import {actions, selectors} from '@neos-project/neos-ui-redux-store';
 import {isNodeCollapsed} from '@neos-project/neos-ui-redux-store/src/CR/Nodes/helpers';
 import {neos} from '@neos-project/neos-ui-decorators';
 
-import {hasNestedNodes} from '@neos-project/neos-ui/src/Containers/LeftSideBar/NodeTree/helpers';
+import {hasNestedNodes} from './../../../../Containers/LeftSideBar/NodeTree/helpers';
```

Comment:
self referencing packages didnt seem to work with webpack...

```
Module not found: Error: Can't resolve '@neos-project/neos-ui/src/Registry' in '/home/macode/untouched/neos-development/Packages/Application/Neos.Neos.Ui/packages/neos-ui/src'
@ ./packages/neos-ui/src/manifest.hotkeys.js 7:16-61
@ ./packages/neos-ui/src/manifest.js
@ ./packages/neos-ui/src/index.js
@ multi ./packages/neos-ui/src/index.js
```

Reply:
also {HotkeyRegistry} cannot be instantiated (because we need to import the default)
but as this worked before - there might be still problems like this in the codebase ...